app/models/katello/glue/candlepin/pool.rb

```diff
@@ -162,27 +164,20 @@ def create_product_associations
       end
 
       def import_hosts
-        uuids = Resources::Candlepin::Pool.consumer_uuids(self.cp_id)
```

Comment:
Refactored this into a lazy_accessor to keep this method more focused (too big imo)